data/block/metaBlock.go

```diff
@@ -392,6 +392,11 @@ func (m *MetaBlock) GetMiniBlockHeaderHandlers() []data.MiniBlockHeaderHandler {
 	return mbHeaderHandlers
 }
 
+// HasScheduledSupport sets the miniBlock headers from the given miniBlock header handlers
```

Comment:
wrong comment, might use the one defined in the shard block

Reply:
done